Tests/Helpers/FeedStoreTestHelper.swift

```
@@ -0,0 +1,31 @@
+//
+//  FeedStoreTestHelper.swift
+//  Tests
+//
+//  Created by Пильтенко Роман on 13.02.2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+
+internal final class FeedStoreTestHelper {
+	internal static func clearTestStoreCache(at url: URL) {
+		let fileManager = FileManager.default
+		do {
+			let directoryContents = try FileManager.default.contentsOfDirectory(at: url,
+																				includingPropertiesForKeys: nil,
+																				options: [])
+			for file in directoryContents {
+				do {
+					try fileManager.removeItem(at: file)
+				}
+				catch let error as NSError {
+					debugPrint("Ooops! Something went wrong: \(error)")
+				}
+				
+			}
+		} catch let error as NSError {
+			print(error.localizedDescription)
+		}
+	}
```

This test helper will delete *all* files in the given directory. In this test target, you're passing the `caches` folder, which is a folder shared with other processes. This can lead to tests affecting other tests and processes. 

Ideally, a test should start in a clean state and finish in a clean state - so it should only delete the files it created.

Moreover, the errors are not handled properly. It just prints the errors, which makes it harder to debug. Errors should be handled.

Check the Realm docs to learn how to delete the Realm files stored to disk (and, check how other students did it).